### PR TITLE
Searchbox component

### PR DESCRIPTION
**Pull request by ChiragdevRobosoft**
Search Component made common

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,13 +10,17 @@
     "@types/jest": "26.0.24",
     "@types/node": "12.20.37",
     "@types/react": "17.0.37",
+    "@types/react-datepicker": "^4.3.4",
```
Comment (naikashok) [inline package.json]: Remove up arrow in all places

File: src/components/common/SearchComponent/index.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+            </Locationholder>
+            <Dateholder className={pageType}>
+              <DateInput className={pageType} />
+            </Dateholder>
+          </LocAndDate>
+        </WrapperHome>
+      ) : (
+        <Wrapper>
+          <ContentContainer>
+            <DeliveryAsap>Delivery ASAP</DeliveryAsap>
```
Comment (naikashok) [inline src/components/common/SearchComponent/index.tsx]: Move hardcoded string

File: src/components/common/dateInput/index.tsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+const DateInput: FC<dateInputProps> = ({ className }) => {
+  let today = moment().format("MMMM DD YYYY");
+  const [newDate, setNewDate]: any = useState("");
+
+  return (
+    <>
+      {className === "home" ? (
+        <Dateholder className={className}>
+          {console.log(newDate)}
```
Comment (naikashok) [inline src/components/common/dateInput/index.tsx]: Remove console.log

File: src/components/common/dateInput/index.tsx [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+            className="datePicker"
+            dateFormat="MMMM dd, yyyy"
+            placeholderText={today}
+            wrapperClassName="datePicker"
+          />
+          <DateBtn className={className} />
+        </Dateholder>
+      ) : (
+        <Dateholder className={className}>
+          {console.log(newDate)}
```
Comment (naikashok) [inline src/components/common/dateInput/index.tsx]: remove console.log

File: package.json (hunk 1 above)
Comment (ChiragdevRobosoft) [reply package.json]: Removed all ^ from json file

File: src/components/common/SearchComponent/index.tsx (hunk 2 above)
Comment (ChiragdevRobosoft) [reply src/components/common/SearchComponent/index.tsx]: Moved hardcoded string to constants.json

File: src/components/common/dateInput/index.tsx (hunk 3 above)
Comment (ChiragdevRobosoft) [reply src/components/common/dateInput/index.tsx]: Removed console.log

File: src/components/common/dateInput/index.tsx (hunk 4 above)
Comment (ChiragdevRobosoft) [reply src/components/common/dateInput/index.tsx]: removed console.log

**Review by naikashok — APPROVED** [review]

**Pull request merged**